### ISSUE TITLE
Update ICS misbehaviour msg definition

### ISSUE THREAD

**Issue by sainoe** [cosmos/interchain-security#1125]

**Comment by sainoe** [comment]
@mpoke agree it isn't a problem, rather a cheap enhancement

**Comment by ancazamfir** [comment]
> @mpoke agree it isn't a problem, rather a cheap enhancement

I agree with @mpoke comment and not sure about the benefit of generalizing ICS only for the misbehaviour aspect.

**Comment by sainoe** [comment]
The issue was decided to be not needed.
